File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/FirePosition.java
package org.firstinspires.ftc.teamcode;

import androidx.annotation.NonNull;

@SuppressWarnings("unused")
public enum FirePosition {
    HIGH,
    MEDIUM,
    LOW,
    ;

    @NonNull
    @Override
    public String toString() {
        if (this == FirePosition.HIGH) {

            return "Fire position: high";
        }
        if (this == FirePosition.MEDIUM) {
            return "Fire position: medium";
        }
        if (this == FirePosition.LOW) {
            return "Fire position: low";
        }
        return "Fire position: unknown";
    }
}

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/FirePosition.kt
package org.firstinspires.ftc.teamcode

enum class FirePosition {
    HIGH, MEDIUM, LOW;

    override fun toString(): String {
        if (this == HIGH) {
            return "Fire position: high"
        }
        if (this == MEDIUM) {
            return "Fire position: medium"
        }
        return if (this == LOW) {
            "Fire position: low"
        } else "Fire position: unknown"
    }
}